pyvista/utilities/transformations.py

```diff
@@ -240,7 +244,10 @@ def apply_transformation_to_points(transformation, points, inplace=False):
     >>> points_orig = points.copy()
     >>> scale_factor = 2
     >>> tf = scale_factor * np.eye(4)
-    >>> tf[3, 3,] = 1
+    >>> tf[
+    ...     3,
+    ...     3,
+    ... ] = 1
```

Comment:
I looked through the 45 changed files and this was the only terrible change. I suspect this is mostly a bug, and has to do with the trailing comma. I'd consider removing this trailing comma in the follow-up PR.

Reply:
Resolved in https://github.com/pyvista/pyvista/pull/4129.